benchmarks/utils/monitor.py

```diff
@@ -183,8 +183,6 @@ def exec_func(*args, **kwargs) -> Dict[str, List[float]]:
             if parametrize is None:
                 # Use the parameters passed by the call
                 parametrization = (call_args, call_kwargs)
-            elif isinstance(parametrize, dict):
-                parametrization = (parametrize, ())
```

Comment:
I'm not quite sure about that, or was it an error with my code? I'd have to re-review.
It is stated in the parameters that:
```
    parametrize : a collection of collections of parameters, optional
        Specifies the parameter matrix to be used on the function.
        These can either be passed as positional arguments (e.g., list of list
        of parameters), or as keyword arguments (e.g., dictionary of list of
        parameters).
        Note: when specified, ignores the parameters passed to the function.
```

Reply:
What do you think @jovan-stojanovic? Do you remember why you removed this part?

Reply:
Yes, actually the code was not working for dictionary of list of parameters. Now it works for all cases!